_posts/2020-03-15-method-s-arguments-pt-1.markdown

```diff
@@ -0,0 +1,373 @@
+---
+layout: post
+title: Ruby Method's Arguments
+date: 2020-03-15 09:30:00
+categories: [ruby, learning]
+author: arieljuod
+---
+
+Ruby is an object oriented language where everything is an object (even methods are objects of the class Method!), so everything we need to do is done by calling methods on objects. That also means that methods have to provide a lot of flexibility because they are used everywhere.
+
+Ruby provides a lot of options to pass arguments to our methods, so we'll make this topic a series so it's not too long. We'll split the options in different categories and then break down everything with some examples and/or use cases.
```

Comment:
```suggestion
Ruby provides a lot of options to pass arguments to our methods, so we'll make this topic a series so it's not too long. We'll split the options into different categories and then break down everything with some examples and/or use cases.
```

Reply:
Maybe you should say something here about this being the second part of the series. I know it's in the title, but because this is the same opening as the previous blog, it might be sightly confusing.